util.js

```diff
@@ -1,84 +1,33 @@
-const aes256 = require('aes256');
-const aesKey = 'ifjkwnfu487348fn3mu4f298m3fm23f923mfn23bf';
-
-const fctIdentityUtils = require('factom-identity-lib/src/crypto');
-
-const {FactomCli} = require('factom');
 
 const {Entry} = require('factom/src/entry');
 const {Chain} = require('factom/src/chain');
 
-const {FactomIdentityManager} = require('factom-identity-lib');
-
-const {FactomdCache} = require('factomd-cache');
 
 const factomCrypto = require('factom-identity-lib/src/crypto');
 
-module.exports.getIssuanceChainId = function (tokenId) {
+//See FATIP-100 Chain ID Derivation Standard
+module.exports.getIssuanceChainId = function (tokenId, rootChainId) {
     return new Chain(Entry.builder()
+        .extId(new Buffer("token"))
```

Comment:
new Buffer is deprecated: https://nodejs.org/api/buffer.html#buffer_new_buffer_array. Use Buffer.from instead.